notebooks/json_blocks/WET/BLOCK-T419.ipynb

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
<p>Just a check to make sure I understand. You take a triplet so that you can estimate the wavefront using the full array mode with the two defocused images and then you compare that answer to what you get from the last in-focus image where you estimate the wavefront from just the corners?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>